### PR TITLE
Changed the communicative functions model since the old one had probl…

### PR DESCRIPTION
**Pull request by marcelmmm**
New communicative functions model

### CODE REVIEW

File: flair/models/text_classification_model.py [hunk 1]
```diff
@@ -478,7 +478,7 @@ def _fetch_model(model_name) -> str:
         
         #Communicative Functions Model
         model_map["communicative-functions"] = "/".join(
-            [hu_path, "comfunc", "communicative-functions-v0.5b.pt"]
+            [hu_path, "comfunc", "communicative-functions.pt"]
```
Comment (alanakbik) [inline flair/models/text_classification_model.py:481]: This should be the new model, right?

**Review by alanakbik — CHANGES_REQUESTED** [review]

File: flair/models/text_classification_model.py (hunk 1 above)
Comment (marcelmmm) [reply flair/models/text_classification_model.py:481]: The new model is called "communicative-function.pt" without the end part "-v0.5b". I admit its confusing.

Comment (oneSebastian) [reply flair/models/text_classification_model.py:481]: This change is indeed still required for the new model to be downloaded.

**Review by alanakbik — APPROVED** [review]

**Comment by alanakbik** [comment]
@marcelmmm finally merged, sorry for waiting so long!

**Pull request merged**